TerminalDocs/customize-settings/profile-settings.md

```diff
@@ -0,0 +1,423 @@
+---
+title: Windows Terminal Profile Settings
+description: Learn how to customize the individual profiles within Windows Terminal.
+author: cinnamon-msft
+ms.author: cinnamon
+ms.date: 05/19/2020
+ms.topic: overview
+ms.service: terminal
+---
+
+# Profile settings in the Windows Terminal
+
+The properties listed below are specific to each unique profile.
+
+## Unique identifier
+
+Profiles can use a GUID as a unique identifier. To make a profile your default profile, it needs a GUID for the `defaultProfile` global setting.
+
+**Property name:** `guid`
+
+**Necessity:** Required
+
+**Accepts:** GUID as a string in registry format: `"{00000000-0000-0000-0000-000000000000}"`
+
+<br />
+
+___
+
+## Executable settings
+
+### Command line
+
+The executable used in the profile.
+
+**Property name:** `commandline`
+
+**Necessity:** Optional
+
+**Accepts:** Executable file name as a string
+
+### Source
+
+Stores the name of the profile generator that originated this profile. _There are no discoverable values for this field._
```

Comment:
Why aren't they discoverable? 

I mean, **I** know why. But I expect someone to wonder why.

Reply:
I'm not sure how to explain why. Terminal generates the stuff in behind the scenes?

Reply:
It might be worth us having a page explaining dynamic profiles. Our users on the repo have expressed some confusion and anger about them :/

Reply:
Then of course we could link to it!

Reply:
Already made a page :) I'll add a link to this description.